crypto/rand_test.go

```diff
@@ -11,6 +11,14 @@ func TestRandIntnPanics(t *testing.T) {
 			t.Error("expected panic for n <= 0")
 		}
 	}()
-	RandIntn(0)
-	RandIntn(-1)
+
+	_, err := RandIntn(0)
+	if err != nil {
+		t.Error("expected panic on n <= 0, not error")
+	}
+
+	_, err = RandIntn(-1)
+	if err != nil {
+		t.Error("expected panic on n <= 0, not error")
+	}
 }
```

Comment:
that could potentially go in the build package, looks like it would work to me, makes checking panics linear instead of a stack that's coded backwards.